pkg/action/cl001/ac000/explainer.go

```diff
@@ -2,19 +2,25 @@ package ac000
 
 import (
 	"context"
+	"time"
 
 	"github.com/giantswarm/awscnfm/pkg/table"
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
+	var d time.Duration
```

Comment:
I start to think if it would be better to generate the plan action as well. All of this should be the same code for each cluster scope anyway.